src/Tizen.NUI/src/internal/Interop/Interop.CanvasView.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
```

Comment:
I think internal files also need to add license statement like others.

Reply:
added license